app/src/lib/stores/app-store.ts

```diff
@@ -7194,6 +7194,35 @@ export class AppStore extends TypedBaseStore<IAppState> {
         changesetData.files[0]
       )
     }
+
+    const { pullRequestState } = this.repositoryStateCache.get(repository)
+    if (pullRequestState === null) {
+      // This shouldn't happen.. we just initialized it.
+      sendNonFatalException(
+        'startPullRequest',
+        new Error(
+          'Failed to start pull request because pull request state was null'
+        )
+      )
+      return
+    }
+
+    const { allBranches, recentBranches } = branchesState
+    const { imageDiffType, hideWhitespaceInHistoryDiff, showSideBySideDiff } =
```

Comment:
>  hideWhitespaceInHistoryDiff

There is one of these for changes, and one for history... maybe there should be one for pr diff or maybe it should be remember on the pr dialog state (not app state). Thoughts?

Reply:
I added a third optionin the diffOptions PR .. I landed on it feeling wierd to use changes/history since neither of them were the same. And it was just easier to make the the third option like the others instead of managing in dialog state. But, I am not strong on that decision. (We could restructure for this and default as false for dialog initialization and then pass it up for all future git calls (every time user changes a file selection or changes base branch).